### ISSUE TITLE
Fix issues with visibly-hidden class

### ISSUE THREAD

**Issue by alisonrae** [PublicisSapient/accessible-ecommerce-demo#242]
Please use the css below for the visibly-hidden class. Currently there is a "visibility: hidden" property on this class that is hiding the fieldset legends from the filters checkboxes.

 .visibly-hidden {
  position: absolute !important;
  height: 1px; width: 1px; 
  overflow: hidden;
  clip: rect(1px 1px 1px 1px); /* IE6, IE7 */
  clip: rect(1px, 1px, 1px, 1px);
}

After this, test that you can remove the aria-labels from the checkboxes. The legend should be announced after the checkbox label. If this is working, please remove the aria-labels from the checkboxes.

**Comment by brianelton** [comment]
@alisonrae There is already a class called sr-only that has the same basic css as you posted above, so instead of updating the visibly-hidden class, I have just changed the few places that it was being used to use sr-only instead.

**Comment by liskulic** [comment]
I've removed the aria-labels from the checkboxes. I also moved the Skip to Products link outside of the <fieldset> because they were being announced as part of the group. Pending PR approval, merge and deploy.
